src/Routes/Devices/DeviceTable.js

```diff
@@ -10,9 +10,14 @@ import CustomEmptyState from '../../components/Empty';
 import { useHistory } from 'react-router-dom';
 import { emptyStateNoFliters } from '../../utils';
 import DeviceStatus from '../../components/Status';
+import RetryUpdatePopover from './RetryUpdatePopover';
 
-const getDeviceStatus = (deviceStatus, isUpdateAvailable) =>
-  deviceStatus === 'UPDATING'
+const getDeviceStatus = (deviceStatus, isUpdateAvailable, DispatcherStatus) =>
+  DispatcherStatus === 'ERROR'
+    ? 'error'
+    : DispatcherStatus === 'UNRESPONSIVE'
+    ? 'unresponsive'
+    : DeviceStatus === 'UPDATING'
```

Comment:
`DeviceStatus` isn't referencing the passed `deviceStatus` parameter. We should change it to `deviceStatus` otherwise it will cause an error